odoo/addons/base/models/res_company.py

```diff
@@ -223,7 +224,8 @@ def install_l10n_modules(self):
         uninstalled_modules = self.uninstalled_l10n_module_ids
         is_ready_and_not_test = (
             not tools.config['test_enable']
-            and (self.env.registry.ready or not self.env.registry._init)
+            and (self.env.registry.ready or not self.env.registry._init) and
+            not getattr(threading.current_thread(), 'testing', False)
```

Comment:
Why is `testing` set when loading the demo data?

Reply:
It's in load from module.py. It's set to true before executing test or installing demo data